### ISSUE TITLE
Scala 2.9.1.RC1 is not supported by sbt 0.7.7

### ISSUE THREAD

**Issue by jrudolph** [sbt/sbt#133]
This happens when trying to compile something for 2.9.1.RC1 in sbt 0.7.7:

```
[info] == console ==
[info] 'compiler-interface' not yet compiled for Scala 2.9.1.RC1. Compiling...
/tmp/sbt_17e60b52/LoggerReporter.scala:27: error: not found: value
countElementsAsString
                       log.warn(Message(countElementsAsString(WARNING.count, "warning") + " found"))
                                        ^
/tmp/sbt_17e60b52/LoggerReporter.scala:29: error: not found: value
countElementsAsString
                       log.error(Message(countElementsAsString(ERROR.count, "error") + " found"))
                                         ^
two errors found
```

**Comment by ijuma** [comment]
This will be fixed in Scala 2.9.1 RC2 (out soon) as I understand.

**Comment by SethTisue** [comment]
see https://issues.scala-lang.org/browse/SI-4838
